ibis/tests/all/conftest.py

```diff
@@ -201,6 +201,11 @@ def geo_df(geo):
     return None
 
 
+@pytest.fixture
+def pandas_testing_client(data_directory):
```

Comment:
Is this being used?

Reply:
it seems I am not using that any more. thanks for catching that.